app/views/layouts/application.html.erb

```diff
@@ -5,12 +5,15 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
```

Comment:
- [ ] Kindly check how you setup tailwind, am getting this error whenever I run your tests:
![image](https://user-images.githubusercontent.com/61408860/187511971-8df5a436-5458-47ad-99ff-45cbb3f8991d.png)
`This leads to failure of your tests`
![image](https://user-images.githubusercontent.com/61408860/187512090-8cfa6f8c-945e-4297-8020-84b7628c0fe8.png)

Reply:
Hello @ppmarq1, I didn't get it on my side

Reply:
Kindly run `bundle exec rake assets:precompile` to solve it in your side
Thanks